docs/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
 
+### 2.26.1
+
+- `Fix`- add popular grammar checker extension node names
```

Comment:
```suggestion
- `Fix`- Fix of Block deletion after Backspace press in case when node cotains 3rd party grammar checker extension
```

and move this line under 2.26.0, please.